runtime/src/runtime/randomness.rs

```diff
@@ -19,3 +21,63 @@ pub enum DomainSeparationTag {
     PoStChainCommit = 9,
     EvmPrevRandao = 10,
 }
+
+pub fn draw_randomness(
```

Comment:
Clippy is convinced this is unused, because it is only called from the FVM-specific code. Is there a more sophisticated way to fix this than slapping allow(unused) on it?

Reply:
Not really.